cpp/src/arrow/compute/kernels/scalar_if_else_benchmark.cc

```diff
@@ -403,36 +499,45 @@ static void ChooseBench64(benchmark::State& state) {
   return ChooseBench<Int64Type>(state);
 }
 
+// IfElse: Numerics
 BENCHMARK(IfElseBench32)->Args({kNumItems, 0});
 BENCHMARK(IfElseBench64)->Args({kNumItems, 0});
-
 BENCHMARK(IfElseBench32)->Args({kNumItems, 99});
 BENCHMARK(IfElseBench64)->Args({kNumItems, 99});
-
 BENCHMARK(IfElseBench32Contiguous)->Args({kNumItems, 0});
 BENCHMARK(IfElseBench64Contiguous)->Args({kNumItems, 0});
-
 BENCHMARK(IfElseBench32Contiguous)->Args({kNumItems, 99});
 BENCHMARK(IfElseBench64Contiguous)->Args({kNumItems, 99});
 
+// IfElse: Lists
+BENCHMARK(IfElseBenchListUInt32)->Args({kNumItems, 0});
+BENCHMARK(IfElseBenchListUInt64)->Args({kNumItems, 0});
+BENCHMARK(IfElseBenchListString32)->Args({kNumItems, 0});
+BENCHMARK(IfElseBenchListString64)->Args({kNumItems, 0});
+BENCHMARK(IfElseBenchListUInt32Contiguous)->Args({kNumItems, 0});
+BENCHMARK(IfElseBenchListUInt64Contiguous)->Args({kNumItems, 0});
+BENCHMARK(IfElseBenchListString32Contiguous)->Args({kNumItems, 0});
+BENCHMARK(IfElseBenchListString64Contiguous)->Args({kNumItems, 0});
```

Comment:
Perhaps we want to cut down on the number of benchmarks? I don't think we need to benchmark both 32- and 64-bit ints, or normal and large strings.